v2/tools/generator/internal/astmodel/interface_type.go

```diff
@@ -200,7 +200,7 @@ func functionToField(codeGenerationContext *CodeGenerationContext, name string,
 		names = []*dst.Ident{dst.NewIdent(name)}
 	}
 
-	f := function.AsFunc(codeGenerationContext, nil) // Empty typename here because we have no receiver
+	f, _ := function.AsFunc(codeGenerationContext, nil) // Empty typename here because we have no receiver
```

Comment:
Worth adding a comment here on why we're ignoring error here?

Reply:
Refactored to **not** ignore the error. 😉